api/Response.json

```diff
@@ -543,6 +543,43 @@
           }
         }
       },
+      "json_static": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Response/json_static",
+          "spec_url": "https://fetch.spec.whatwg.org/#dom-response-json-data-init-init",
```

Comment:
Note, I chose this link because it takes you to the correct line in the IDL. There is a URL which takes you to the implementation steps, but IMO the IDL is the best starting point link